### ISSUE TITLE
Unable to install latest version from PSGallery

### ISSUE THREAD

**Issue by mohandatla** [PowerShell/PowerShellGet#126]
I can not activate the below issue, creating a new issue to follow up on this.
Workaround is to unblock, but this needs to be fixed correctly, 
https://github.com/PowerShell/PowerShellGet/issues/124

**Comment by palchak** [comment]
I am not able to install Powershell Get, if i run the command 

Save-Module -Name PowerShellGet -Path C:\Pallab\Test  -Repository PSGallery i get the below error:

PackageManagement\Get-PackageProvider : The term 'PackageManagement\Get-PackageProvider' is not recognized as the 
name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was 
included, verify that the path is correct and try again.
At C:\Users\palchakraborty\Documents\WindowsPowerShell\Modules\PowerShellGet\src\PowerShellGet\private\functions\Inst
all-NuGetClientBinaries.ps1:41 char:26
+ ...        $nugetProvider = PackageManagement\Get-PackageProvider -ErrorA ...
+                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (PackageManagement\Get-PackageProvider:String) [], CommandNotFoundExce 
   ption
    + FullyQualifiedErrorId : CommandNotFoundException
 
PackageManagement\Get-PackageProvider : The term 'PackageManagement\Get-PackageProvider' is not recognized as the 
name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was 
included, verify that the path is correct and try again.
At C:\Users\palchakraborty\Documents\WindowsPowerShell\Modules\PowerShellGet\src\PowerShellGet\private\functions\Inst
all-NuGetClientBinaries.ps1:55 char:40
+ ... ailableNugetProviders = PackageManagement\Get-PackageProvider -Name $ ...
+                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (PackageManagement\Get-PackageProvider:String) [], CommandNotFoundExce 
   ption
    + FullyQualifiedErrorId : CommandNotFoundException
 
PackageManagement\Install-PackageProvider : The term 'PackageManagement\Install-PackageProvider' is not recognized 
as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path 
was included, verify that the path is correct and try again.
At C:\Users\palchakraborty\Documents\WindowsPowerShell\Modules\PowerShellGet\src\PowerShellGet\private\functions\Inst
all-NuGetClientBinaries.ps1:243 char:21
+             $null = PackageManagement\Install-PackageProvider -Name $ ...
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (PackageManageme...PackageProvider:String) [], CommandNotFoundExceptio 
   n
    + FullyQualifiedErrorId : CommandNotFoundException
 
PackageManagement\Import-PackageProvider : The term 'PackageManagement\Import-PackageProvider' is not recognized as 
the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was 
included, verify that the path is correct and try again.
At C:\Users\palchakraborty\Documents\WindowsPowerShell\Modules\PowerShellGet\src\PowerShellGet\private\functions\Inst
all-NuGetClientBinaries.ps1:249 char:21
+             $null = PackageManagement\Import-PackageProvider -Name $s ...
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (PackageManagement\Import-PackageProvider:String) [], CommandNotFoundE 
   xception
    + FullyQualifiedErrorId : CommandNotFoundException
 
PackageManagement\Get-PackageProvider : The term 'PackageManagement\Get-PackageProvider' is not recognized as the 
name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was 
included, verify that the path is correct and try again.
At C:\Users\palchakraborty\Documents\WindowsPowerShell\Modules\PowerShellGet\src\PowerShellGet\private\functions\Inst
all-NuGetClientBinaries.ps1:253 char:30
+ ...        $nugetProvider = PackageManagement\Get-PackageProvider -Name $ ...
+                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (PackageManagement\Get-PackageProvider:String) [], CommandNotFoundExce 
   ption
    + FullyQualifiedErrorId : CommandNotFoundException
 
Save-Module : NuGet provider is required to interact with NuGet-based repositories. Please ensure that '2.8.5.201' 
or newer version of NuGet provider is installed.
At line:1 char:1
+ Save-Module -Name PowerShellGet -Path C:\Pallab\Test  -Repository PSG ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Save-Module], InvalidOperationException
    + FullyQualifiedErrorId : CouldNotInstallNuGetProvider,Save-Module
